azure-pipelines.yml

```diff
@@ -60,10 +60,13 @@ stages:
               conda env create -f environment-dev.yml
               source activate foyer-dev
               pip install -e .
+              cd src/antefoyer
+              pip install -e .
+              cd ../../
```

Comment:
```suggestion
```
I don't think this step is necessary. I think each bash step starts in whatever the root dir is, in this case foyer.

Reply:
Reviewing it a bit more, i think you are correct. That does seem to be the case, it will default for each `step` `$Build.SourcesDirectory`